openapi/zuora.collection/Ballerina.toml

```
@@ -0,0 +1,9 @@
+[package]
+org = "ballerinax"
+name = "zuora.collection"
+version = "1.0.0-SNAPSHOT"
```

Shall we remove the `-SNAPSHOT` part from the version?
Refer https://github.com/ballerina-platform/ballerina-extended-library/issues/134
```suggestion
version = "1.0.0"
```